Makefile

```diff
@@ -2,7 +2,7 @@ obj-m += acer-wmi-battery.o
 PWD := $(CURDIR)
 
 all:
-	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules
+	make -C /lib/modules/$(KERNELVERSION)/build M=$(PWD) modules
```

Comment:
If you need to be able to pass a kernel version to the Makefile, it would be better to make this optional, i.e. specify a default value for `KERNELVERSION`. This can be accomplished by defining the variable at the top of the Makefile like this:
`KERNELVERSION ?= "$(shell uname -r)"`.